src/wmnds/patterns/beta-banner/_beta-banner.njk

```diff
@@ -0,0 +1,13 @@
+{% macro wmndsBetaBanner(params) %}
+
+<div class="wmnds-grid banner-container">
+<div class="wmnds-col-1-8">
+<a href="/" title="West Midlands Network Design System Alpha" class="wmnds-phase-indicator">
+        beta
+      </a>
```

Comment:
Import the phase indicator via the nunjucks syntax....this means it will always have the most up-to-date version of the component.
```suggestion
{% from "wmnds/components/phase-indicator/_phase-indicator.njk" import wmndsPhaseIndicator %}
{{wmndsPhaseIndicator()}}
```

Reply:
I want it to display Beta but if I make the change suggested it replaces it with alpha.

Reply:
Include it for now, and I will change it to beta in the future.
It's just a placeholder for now.